manifests/gcp_marketplace/guide.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Kubeflow Pipelines from Google Cloud Marketplace
+
+You can go to [Google Cloud Marketplace](https://console.cloud.google.com/marketplace) to deploy Kubeflow Pipelines via UI.
+To get to Kubeflow Pipelines page in Marketplace, enter "Kubeflow Pipelines" in the search box in Marketplace landing page. 
+Alternatively you can access the [Marketplace page for Kubeflow Pipelines](https://console.cloud.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines) directly.
+
+You can find Kubeflow Pipelines instances deployed via Marketplace in [Pipelines Console](http://console.cloud.google.com/ai-platform/pipelines).
+
+## Cluster
+
+You can choose an existing cluster for Kubeflow Pipelines deployment, or create a new cluster. New cluster created from Marketplace config page cannot be customized.
```

Comment:
You can deploy Kubeflow Pipelines to a new cluster or an existing cluster. New clusters aren't customizable, so if you need that, you should use the [Google Kubernetes Engine console](https://console.cloud.google.com/kubernetes/list) to create a cluster that meets your requirements and then deploy to that.